lib/ui/toolbar.jsx

```diff
@@ -469,7 +469,7 @@ export default class Toolbar {
 		this.targetOptions = [];
 		this.targetOptions.push({ value: '', text: 'Devices', disabled: true });
 
-		const selectedTarget = atom.config.get(`appcelerator-titanium.general.${this.normalizedAppName}.ios.selectedTarget`);
+		const selectedTarget = Utils.get(`appcelerator-titanium.general.${this.normalizedAppName}.ios.selectedTarget`);
```

Comment:
are those `appcelerator-titanium` strings still correct?

Reply:
Ha, and here was me thinking I got them all! They'll be handled by the regex in the function. I need to fix the function name anyway so I'll fix these